index.html

```diff
@@ -627,6 +627,70 @@ <h2>DID Documents</h2>
+representation of a resource for a given <a>DID URL</a>.
+      </p>
+      <p>
+<a>DID resolution</a> and <a>DID URL dereferencing</a> are defined as abstract functions
+(see Section <a href="#resolving-a-did"></a> and Section <a href="#dereferencing-a-did-url"></a>).
+      </p>
+      <p>
+These abstract functions are implemented by <a>DID resolvers</a>. A <a>DID resolver</a> is invoked
+by a <a>client</a> via a <a>binding</a>. Bindings define how the abstract functions are realized
+using concrete programming or communication interfaces. It is possible to distinguish between
+<a>local bindings</a> and <a>remote bindings</a>.
+      </p>
+      <dl>
+        <dt><dfn>binding</dfn></dt>
+        <dd>A concrete mechanism through which a <a>client</a> invokes a <a>DID resolver</a>. This could be a <a>local binding</a> such as a local command line tool or library API, or a <a>remote binding</a> such as an HTTP(S) binding.</dd>
+      </dl>
+      <dl>
+        <dt><dfn>client</dfn></dt>
+        <dd>Software and/or hardware that invokes a <a>DID resolver</a> in order to execute the <a>DID resolution</a> and/or <a>DID URL dereferencing</a> functions. This invocation is done via a <a>binding</a>. The term <a>client</a> does not imply any specific network topology.</dd>
+      </dl>
+      <dl>
+        <dt><dfn>local binding</dfn></dt>
+        <dd>A <a>binding</a> where the <a>client</a> invokes a <a>DID resolver</a> that runs on the same network host, e.g. via a local command line tool or library API.
+        In this case, the <a>DID resolver</a> is sometimes also called a "local <a>DID resolver</a>".
+      </dl>
+      <dl>
+        <dt><dfn>remote binding</dfn></dt>
+        <dd>A <a>binding</a> where the <a>client</a> invokes a <a>DID resolver</a> that runs on a different network host, e.g. via an HTTP(S) binding.
+        In this case, the <a>DID resolver</a> is sometimes also called a "remote <a>DID resolver</a>".
+      </dl>
+      <p>
+        <figure id="figure-binding-local">
+          <img style="margin: auto; display: block; width: 100%;" src="diagrams/binding-local.png"
+            alt="Diagram showing a DID Resolver with a 'local binding'.">
+          <figcaption style="text-align: center;">
+A <a>local binding</a> for a <a>DID resolver</a>.
+          </figcaption>
+        </figure>
+        <figure id="figure-binding-remote">
+          <img style="margin: auto; display: block; width: 100%;" src="diagrams/binding-remote.png"
+            alt="Diagram showing a DID resolver with a 'remote binding'.">
+          <figcaption style="text-align: center;">
+A <a>remote binding</a> for a <a>DID resolver</a>.
+          </figcaption>
+        </figure>
+      </p>
+
+      <p>
+Additional considerations about architectures of the <a>DID resolution</a> and <a>DID URL
+dereferencing</a> functions, including concepts such as "proxied resolution" or "client-side
+dereferencing", can be found in [[DID-RESOLUTION]].
```

Comment:
```suggestion
dereferencing", can be found in [[?DID-RESOLUTION]].
```

Reply:
Thanks @brentzundel, just to clarify, do you propose to make this change only here, or at every occurrence of [[DID-RESOLUTION]] (there are a few more)?